src/blockdev.rs

```diff
@@ -829,6 +879,55 @@ pub fn lsblk(dev: &Path, with_deps: bool) -> Result<Vec<HashMap<String, String>>
     Ok(result)
 }
 
+pub fn find_esps() -> Result<Vec<String>> {
+    const ESP_TYPE_GUID: &str = "c12a7328-f81f-11d2-ba4b-00a0c93ec93b";
+    let mut cmd = Command::new("lsblk");
+    // Older lsblk, e.g. in CentOS 7.6, doesn't support PATH, but --paths option
+    cmd.arg("--pairs")
+        .arg("--paths")
+        .arg("--output")
+        .arg("NAME,PARTTYPE");
+    let output = cmd_output(&mut cmd)?;
+    output
+        .lines()
+        .filter_map(|line| {
+            let dev = split_lsblk_line(line);
+            if dev.get("PARTTYPE").map(|t| t.as_str()) == Some(ESP_TYPE_GUID) {
```

Comment:
Another `.as_deref()` candidate.

Reply:
Heh, I've also tried this at first, but

```
error[E0308]: mismatched types
   --> src/blockdev.rs:895:55
    |
895 |             if dev.get("PARTTYPE").as_deref() == Some(ESP_TYPE_GUID) {
    |                                                       ^^^^^^^^^^^^^ expected struct `std::string::String`, found `str`
    |
    = note: expected reference `&std::string::String`
               found reference `&'static str`
```

I think it's because we're not dealing with an `Option<String>`, but an `Option<&String>`.